packages/fast-components-react-base/src/horizontal-overflow/horizontal-overflow.tsx

```diff
@@ -377,7 +377,20 @@ class HorizontalOverflow extends Foundation<
         return React.Children.map(
             this.withoutSlot([ButtonDirection.previous, ButtonDirection.next]),
             (child: React.ReactNode): React.ReactElement<HTMLLIElement> => {
-                return <li style={{ display: "inline-block" }}>{child}</li>;
+                const itemClass: string = get(
+                    this.props.managedClasses,
+                    "horizontalOverflow_item",
+                    ""
+                );
+
+                return (
+                    <li
+                        className={itemClass}
+                        style={itemClass === "" ? { display: "inline-block" } : null}
```

Comment:
Presumably we would still want these to display inline-block though, wouldn't' we? Does the component work without setting this?

My guess is you're making this conditional because you don't want specificity issues, right?

Reply:
Exactly - There's an assumption here that would need to be documented that you are "taking active control" by passing a class. With inline-block here, you must include "!important" to change this.

Reply:
> that would need to be documented

Can we update the documentation then? It's not apparent why we are doing it this way from glancing at the code.

Reply:
The comment was from the initial draft PR form, which is why it was there and not as part of the PR. Regardless, this is outdated; ppl can use important to override the position.